packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -861,7 +869,7 @@ const DatePickerInput = (props) => {
           onBlur={onBlur}
           onFocus={onButtonFocus}
           refCallback={buttonRefCallback}
-          aria-label={`${calendarDate} ${intl.formatMessage({ id: 'Terra.datePicker.openCalendar' })}`}
```

Comment:
If this is removed, then SR will not announce the date selected from calendar popup upon closure. This needs to discussed with a11y team. This date is to provide date context due to instant closing of calendar.